### PR TITLE
Add time_zone_info as a choice for time zone support

### PR DESCRIPTION
**Pull request by mathieuprog**

### CODE REVIEW

**Review by whatyouhide — COMMENTED** [review]
I've never used time_zone_info. Do you know if it's used in production somewhere, and how "field tested" it is? 🙃

(Looks good to me btw, just curious!)

File: lib/elixir/lib/calendar/datetime.ex [hunk 1]
```diff
@@ -38,6 +38,7 @@ defmodule DateTime do
   available options and libraries:
 
     * [`tz`](https://github.com/mathieuprog/tz)
+    * [`time_zone_info`](https://github.com/hrzndhrn/time_zone_info)
```
Comment (josevalim) [inline lib/elixir/lib/calendar/datetime.ex:41]:
```suggestion
    * [`time_zone_info`](https://github.com/hrzndhrn/time_zone_info)
    * [`tz`](https://github.com/mathieuprog/tz)
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by mathieuprog** [comment]
@whatyouhide it is the only library besides tz that has no bugs regarding time zone operations on dates. See https://github.com/mathieuprog/tzdb_test with comparison on correctness and performance.

**Pull request merged**